.travis.yml

```diff
@@ -17,10 +17,11 @@ jobs:
       script: |
         clang-format-7 --version
         # build a pathspec that excludes the files in .clang-format-ignore
-        while read file ; do echo EXCLUDES+="':(top,exclude)$file' " ; done < .clang-format-ignore
+        while read file ; do EXCLUDES+="':(top,exclude)$file' " ; done < .clang-format-ignore
         MERGE_BASE=$(git merge-base ${TRAVIS_COMMIT_RANGE%...*} ${TRAVIS_COMMIT_RANGE#*...})
         echo "Checking for formatting errors introduced since $MERGE_BASE"
-        git-clang-format-7 --binary clang-format-7 $MERGE_BASE -- $EXCLUDES
+        # use eval so that quotes are not interpreted as parts of file names
```

Comment:
clarify that we're talking about quotes that this script inserted into `EXCLUDES`

Reply:
Done